packages/runtime/src/account.ts

```diff
@@ -199,6 +212,18 @@ export class AccountStore implements AccountStoreI {
     checkAndSetASAFields(fields, asset);
   }
 
+  /**
+   * removes asset holding from account
+   * @param assetId asset index
+   */
+  closeAsset (assetId: number): void {
+    if (this.assets.has(assetId)) {
+      this.minBalance -= ASSET_CREATION_FEE;
+      // https://developer.algorand.org/docs/reference/transactions/#asset-transfer-transaction
+      this.assets.delete(assetId); // remove asset holding from account
+    }
```

Comment:
should we throw error if asset is not present in account ? or a warning ?

Reply:
before closing an asset(removing asset holding) we need to empty it's holding to the `closeRemTo` account (even if amount is zero). So we're verifying asset holding exists in `runtime.transferAsset` (before emptying balance). That's why i kept this code there only. Will add a comment here now.